argocd_observability/deployment_event.md

```diff
@@ -0,0 +1,37 @@
+### ArgoCD Sync Deployment event
+
+| Property | Source | in Dashboard used | 
+|---|---|---|
+| event.id  |   |
+| event.kind = "SDLC_EVENT" |   | x 
+| event.version = "0.1.0" | |  x
+| event.category = "task" | |  x
+| event.type = "deployment" | |  x
+| event.provider = "argocd" | |  x
+| event.status | |  x
+| duration | |  x
+| start_time | |  x
+| end_time | |  x
+| timestamp | |  x
+|---|---|---|
+| task.id | |
+| task.name | |
+| task.outcome | | x
+|---|---|---|
+| deployment.service.uid | | x
+| deployment.service.name | | x
+| deployment.service.namespace | | -
+| deployment.service.resource_version | | -
+| deployment.service.labels | | -
+| deployment.target.namespace | | x
+| deployment.target.server.url | | x
+| deployment.images | | -
+|---|---|---|
+| vcs.repository.name | | x
```

Comment:
Although unlikely, I'd rather use the full URL. The name may not be unique

Reply:
My mistake. In fact, it is the `full.url`. I will change it.